datafusion/expr/src/udf.rs

```diff
@@ -305,14 +305,11 @@ pub trait ScalarUDFImpl: Debug + Send + Sync {
     /// value for `('foo' | 'bar')` as it does for ('foobar').
     fn return_type_from_exprs(
         &self,
-        args: &[Expr],
-        schema: &dyn ExprSchema,
+        _args: &[Expr],
+        _schema: &dyn ExprSchema,
+        arg_types: &[DataType],
```

Comment:
I reviewed the documentation of this API, and I think in general it is clear and no changes are needed